src/cmd/cli/command/commands_test.go

```diff
@@ -138,26 +138,27 @@ func TestVersion(t *testing.T) {
 }
 
 func TestCommandGates(t *testing.T) {
-	mockService := &mockFabricService{canIUseResponse: defangv1.CanIUseResponse{}}
+	mockService := &mockFabricService{}
 	_, handler := defangv1connect.NewFabricControllerHandler(mockService)
 
 	server := httptest.NewServer(handler)
 	t.Cleanup(server.Close)
 
-	type cmdPermTest struct {
+	t.Setenv("AWS_REGION", "us-test-2")
+
+	if err := os.Chdir("../../../testdata/empty"); err != nil {
+		t.Fatalf("Failed to change directory: %v", err)
+	}
+
+	testData := []struct {
 		name          string
 		command       []string
 		accessAllowed bool
 		wantError     string
-	}
-	type cmdPermTests []cmdPermTest
-
-	t.Setenv("AWS_REGION", "us-test-2")
-
-	testData := cmdPermTests{
+	}{
 		{
 			name:          "compose up - aws - no access",
-			command:       []string{"compose", "up", "--project-name=app", "--provider=aws", "--dry-run"},
+			command:       []string{"compose", "up", "--provider=aws", "--dry-run"},
```

Comment:
What's the significance of removing the `--project-name` flag here?

Reply:
`up` with `--project-name` made little sense. It wants an actual project.